pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -396,7 +393,7 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 		obj.HairpinMode = PromiscuousBridge
 	}
 	if obj.EvictionHard == nil {
-		temp := "memory.available<100Mi"
+		temp := "memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"
```

Comment:
We could use a literal value here as well if you want.  If we use 256MB, I have a couple questions:
Should we enable inode eviction?  If so, is a percentage still appropriate?
Should I still remove the custom defaults for vagrant, since they used nodefs.available<10% previously?